docs/unique/agent-reflections-and-prompt-evolution.md

```
@@ -235,28 +235,28 @@
-- [Prompt_Folder_Bootstrap](prompt-folder-bootstrap.md)
+- [[prompt-folder-bootstrap|Prompt_Folder_Bootstrap]]
 - [Promethean Event Bus MVP v0.1](promethean-event-bus-mvp-v0-1.md)
 - [Canonical Org-Babel Matplotlib Animation Template](canonical-org-babel-matplotlib-animation-template.md)
-- [Chroma Toolkit Consolidation Plan](chroma-toolkit-consolidation-plan.md)
-- [ecs-scheduler-and-prefabs](ecs-scheduler-and-prefabs.md)
+- [[chroma-toolkit-consolidation-plan|Chroma Toolkit Consolidation Plan]]
+- [[ecs-scheduler-and-prefabs]]
 
 ## Sources
-- [Prompt_Folder_Bootstrap — L113](prompt-folder-bootstrap.md#L113) (line 113, col 3, score 0.86)
+- [[prompt-folder-bootstrap#L113|Prompt_Folder_Bootstrap — L113]] (line 113, col 3, score 0.86)
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)